### PR TITLE
Add Theta.tv

### PR DESCRIPTION
**Pull request by soodaayush**
Unfortunately, I couldn't find an SVG for this website.

### CODE REVIEW

File: entries/t/theta.tv.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "Theta.tv": {
+    "domain": "theta.tv",
```
Comment (hkamran80) [inline entries/t/theta.tv.json:3]:
```suggestion
    "domain": "theta.tv",
    "img": "theta.tv.png",
```

The `img` key is required if the image is not an SVG

**Review by hkamran80 — CHANGES_REQUESTED** [review]
All PNGs should be a max of 128px by 128px and run through an optimizing utility like [TinyPNG](https://tinypng.com). Please read [the guidelines](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#guidelines) for more information.

**Comment by soodaayush** [comment]
Updated the image.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
There's a lot of padding around that image which will make it look smaller than all the other sites listed on 2FA Directory. Try and get it to 128x128 without the padding.

**Comment by soodaayush** [comment]
Is the new image good?

**Review by Carlgo11 — APPROVED** [review]

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**